fighthealthinsurance/management/commands/generate_blog_metadata.py

```
@@ -41,22 +41,34 @@ def handle(self, *args, **options):
                     if m:
                         try:
                             frontmatter = yaml.safe_load(m.group(1)) or {}
-                        except Exception as e:
+                        except yaml.YAMLError as e:
                             self.stderr.write(
                                 self.style.ERROR(
                                     f"Failed to parse frontmatter in {fname}: {e}"
                                 )
                             )
                             continue
+                        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)